src/improvements/template/OPCMUpgradeV200.sol

```diff
@@ -143,9 +143,21 @@ contract OPCMUpgradeV200 is OPCMBaseTask {
             // DF-30: Dispute factory owner must be l1PAOMultisig - It is checking for the OP Sepolia PAO.
             string memory expectedErrors_84532 =
                 "PROXYA-10,DF-30,PDDG-50,PDDG-DWETH-30,PDDG-DWETH-40,PDDG-ANCHORP-40,PDDG-120,PLDG-50,PLDG-DWETH-30,PLDG-DWETH-40,PLDG-ANCHORP-40";
+
+            // Mainnet errors
+            string memory expectedErrors_10 =
+                "L1SB-10,L721B-10,PORTAL-10,PDDG-20,PDDG-DWETH-10,PDDG-DWETH-30,PDDG-DWETH-40,PDDG-ANCHORP-10,PDDG-ANCHORP-40,PDDG-PIMGO-10,PLDG-20,PLDG-DWETH-10,PLDG-DWETH-30,PLDG-DWETH-40,PLDG-ANCHORP-10,PLDG-ANCHORP-40,PLDG-PIMGO-10";
+            string memory expectedErrors_1868 =
+                "SYSCON-20,L1SB-10,L721B-10,PORTAL-10,PDDG-20,PDDG-DWETH-10,PDDG-DWETH-40,PDDG-ANCHORP-10,PDDG-ANCHORP-40,PDDG-PIMGO-10,PLDG-10";
+            string memory expectedErrors_57073 =
+                "SYSCON-30,L1SB-10,L721B-10,PORTAL-10,PDDG-20,PDDG-DWETH-10,PDDG-DWETH-40,PDDG-ANCHORP-10,PDDG-ANCHORP-40,PDDG-PIMGO-10,PLDG-20,PLDG-DWETH-10,PLDG-DWETH-40,PLDG-ANCHORP-10,PLDG-ANCHORP-40,PLDG-PIMGO-10";
+            string memory expectedErrors_8453 =
```

Comment:
You should only need to add `expectedErrors_8453` in this PR right?

Reply:
Let's have a conversation once I have finished all the validations etc..